build-support/javascriptstyle/README.md

```diff
@@ -0,0 +1,62 @@
+# Setting up JavaScript Style Checker
+
+1. Download the JavaScriptStyle package from https://github.com/pantsbuild/binaries/build-support/scripts/javascriptstyle
+2. Unpack the javascriptstyle.tgz into your pants support directory
+3. Configure pants.ini
+4. Add new rules
+5. Add plugins to extend beyond the default ruleset
+6. Blacklisting
+
+## Configuring pants.ini
+
+	[lint.javascriptstyle]
+	javascriptstyle_dir: %(pants_supportdir)s/javascriptstyle
+	skip: False
```

Comment:
Because you don't pass a default for the `--skip` flag, it defaults to `False`. So no need to specify that here.